quantized_mesh_encoder/extensions.py

```diff
@@ -0,0 +1,13 @@
+from dataclasses import dataclass
+
+from .constants import QuantizedMeshExtensions
+
+
+@dataclass
+class ExtensionBase:
+    extension_id: int
```

Comment:
I think it's fine to just use `id: int`

Reply:
Also this base class should be designated with `abc` similar to https://github.com/geospatial-jeff/aiocogeo/blob/5a1d32c3f22c883354804168a87abb0a2ea1c328/aiocogeo/compression.py#L15-L29.

Then we can designate an `encode` method as an `@abc.abstractmethod`, so that every subclass is required to implement the `encode` method.